sdk/core/azure-core/azure/core/pipeline/policies/_authentication.py

```diff
@@ -174,6 +174,10 @@ def __init__(
     ) -> None:
         super(AzureKeyCredentialPolicy, self).__init__()
```

Comment:
micro nit: since we are here, can we just make this `super().__init__()`

Reply:
done